.gitignore

```diff
@@ -26,4 +26,4 @@ Dockerfile.cross
 *~
 
 # Redfish schema bundle files
-hack/DSP8010_*
+hack/DSP8010_*
```

Comment:
Drop this change

Reply:
Still valid :) We don't need any changes in this file, keep the empty new line at the end of the file.